sql/moz-fx-data-shared-prod/mozilla_org_derived/gclid_conversions_v2/query.sql

```diff
@@ -33,18 +36,61 @@ dl_token_to_telemetry_id AS (
   FROM
     `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
 ),
-telemetry_id_to_activity AS (
+--Step 4: Get all clients who were active on the activity date, and the type of activity they had
+telemetry_id_to_activity_staging AS (
   SELECT
     client_id AS telemetry_client_id,
     submission_date AS activity_date,
     search_count_all > 0 AS did_search,
```

Comment:
Could this value be null? Then you could use IFNULL or COALESCE?

Reply:
Good catch, thank you @lucia-vargas-a ! :)